pym/bob/cmds/jenkins.py

```diff
@@ -1091,6 +1095,8 @@ def _genJenkinsJobs(step, jobs, nameCalculator, archiveBackend, seenPackages, al
                 _genJenkinsJobs(sandboxStep, jobs, nameCalculator, archiveBackend,
                                 seenPackages, allVariantIds, shortdescription)
 
+        return jj
```

Comment:
For consistency we should always return `jj`. Otherwise we're implicitly returning `None` for checkout- and build-steps. It doesn't matter at the moment but better not add a pit-fall.